### DISCUSSION TITLE
Nuncabola port to LWJGL 3?

### DISCUSSION THREAD

**Discussion by ghost** [Neverball/neverball#350] [Q&A]
Hello. Why Nuncabola written in old LWJGL? I beg to rewrite to newer version...

**Comment by fwp** [comment]
Work on a new version of Nuncabola that uses LWJGL 3 has already started, but there are still a lot of details that need to be figured out to ensure everything keeps functioning correctly.

(Back in 2009 when Nuncabola was started, only LWJGL 2 existed.)

**Comment by ghost** [comment]
Nuncabola have no Github repo?
